### PR TITLE
S3 objectstore ci

### PR DESCRIPTION
**Pull request by icewind1991**
- [x] requires #768

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Fails:

```
1) OCA\Files\Tests\Service\TagServiceTest::testUpdateFileTags
563s
110
OCP\Files\StorageNotAvailableException: 
563s
112
/drone/src/github.com/nextcloud/server/lib/private/Files/Storage/Wrapper/Availability.php:76
563s
113
/drone/src/github.com/nextcloud/server/lib/private/Files/Storage/Wrapper/Availability.php:239
563s
114
/drone/src/github.com/nextcloud/server/lib/private/Files/View.php:1293
563s
115
/drone/src/github.com/nextcloud/server/lib/private/Files/View.php:1339
563s
116
/drone/src/github.com/nextcloud/server/lib/private/Files/Node/Root.php:196
563s
117
/drone/src/github.com/nextcloud/server/lib/private/Files/Node/Folder.php:122
563s
118
/drone/src/github.com/nextcloud/server/lib/private/Files/AppData/AppData.php:105
563s
119
/drone/src/github.com/nextcloud/server/lib/private/Preview/Watcher.php:61
563s
120
/drone/src/github.com/nextcloud/server/lib/private/Preview/WatcherConnector.php:60
563s
121
/drone/src/github.com/nextcloud/server/lib/private/Hooks/EmitterTrait.php:99
563s
122
/drone/src/github.com/nextcloud/server/lib/private/Hooks/PublicEmitter.php:33
563s
123
/drone/src/github.com/nextcloud/server/lib/private/Files/Node/Root.php:134
563s
124
/drone/src/github.com/nextcloud/server/lib/private/Files/Node/Folder.php:172
563s
125
/drone/src/github.com/nextcloud/server/apps/files/tests/Service/TagServiceTest.php:97
```

**Comment by MorrisJobke** [comment]
@icewind1991 Required PR is merged ;) Fix this! 😉

**Comment by MorrisJobke** [comment]
@icewind1991 Ping

**Comment by LukasReschke** [comment]
@icewind1991 Can we move that to 12? I'd prefer to consider 11 closed for any non-critical merges.

**Comment by rullzer** [comment]
@icewind1991 care to finish this? I'd love to have tests in place.

**Comment by MorrisJobke** [comment]
@icewind1991 Ping ;)

**Comment by icewind1991** [comment]
Changed this so it only includes the object store tests themselves, and not run all tests with object store configured.

That way we can at least get this tests merged and I'll open a separate issue/pr for the rest

cc @MorrisJobke @LukasReschke @rullzer please review

File: autotest.sh [hunk 1]
```diff
@@ -167,7 +167,7 @@ function execute_tests {
 	git checkout tests/data
 
 	# reset data directory
-	rm -rf "$DATADIR"
+	rm -rf "$DATADIR"f
```
Comment (MorrisJobke) [inline autotest.sh]: Typo?

Comment (icewind1991) [inline autotest.sh]: fixed

**Comment by rullzer** [comment]
I rebased to make CI happy

**Review by rullzer — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**